castle/cms/commands.py

```diff
@@ -195,54 +195,34 @@ def __call__(self, filepath):
     logger.warn('gs not installed. Some metadata might remain in PDF files.')
 
 
-class MD5SubProcess(BaseSubProcess):
+class SHA256SubProcess(BaseSubProcess):
     """
-    To get md5 hash of files on the filesystem so
+    To get sha256 encryption of files on the filesystem so
     large files do not need to be loaded into
     memory to be checked
     """
     if os.name == 'nt':
-        bin_name = 'md5.exe'
+        bin_name = 'shasum.exe'
     else:
-        bin_name = 'md5'
+        bin_name = 'shasum'
```

Comment:
isn't it "sha256sum" ? using just `shasum` with no parameters might default to sha1